### PR TITLE
Changed autofill to Google Chromes autofill colour.

### PR DESCRIPTION
**Pull request by will-shaw**
Password is now removed from field when 'remember' is disabled.
Stopped login view printing the eVision mess to stdout.

### CODE REVIEW

File: WingIt/LoginViewController.swift [hunk 1]
```diff
@@ -184,12 +192,11 @@ class LoginViewController: UIViewController, UIWebViewDelegate, UITextFieldDeleg
             if (header == "" && !once) {
                 errorLabel.text = "Enter your eVision details"
                 displayLoginFields()
-                if PWIsStored{
+                if PWIsStored {
                     usernameField.text = retrieveStoredUsername()
-                    usernameField.backgroundColor = UIColor.yellow
```
Comment (BlueDrink9) [inline WingIt/LoginViewController.swift:189]: Rename this autoFillColor

**Review by BlueDrink9 — CHANGES_REQUESTED** [review]
Chrome doesn't remove password from field when remember is disabled. No other site does this. Revert that change.

File: WingIt/LoginViewController.swift (hunk 1 above)
Comment (will-shaw) [reply WingIt/LoginViewController.swift:189]: If you want to be pedantic about variable names then we should have been from the start, but fair enough haha.

Comment (will-shaw) [reply WingIt/LoginViewController.swift:189]: This is fixed and also the above.
Was done from linux however so should make sure to wait for Travis to test the build.

**Review by BlueDrink9 — APPROVED** [review]

**Pull request merged**